packages/algob/src/builtin-tasks/compile.ts

```diff
@@ -25,15 +26,15 @@ export interface TaskArgs {
 }
 
 function _compile ({ force }: TaskArgs, env: AlgobRuntimeEnv): Promise<void> {
```

Comment:
It would be best to rename this function.

Reply:
ah, I forgot it. OK. will do as discussed in the previous PR.